dspace-api/src/test/java/org/dspace/content/packager/DSpaceAIPIntegrationTest.java

```diff
@@ -0,0 +1,1001 @@
+/**
```

Comment:
@mhwood - I do agree with this in concept. Unfortunately though, our existing Test Framework doesn't seem to support this. Simply renaming the class to "DSpaceAIPDisseminateIngestIT" seems to cause it to be entirely ignored so it never gets executed.

So, as discussed in IRC, it seems like we'd need further rework / cleanup of our Test Framework to support this idea. Currently all our other Integration tests seem to be named "*IntegrationTest", e.g. `AbstractIntegrationTest`, `CommunityCollectionIntegrationTest`.